### PR TITLE
WIP: Add classes option

### PR DESCRIPTION
**Pull request by apsdehal**

### CODE REVIEW

File: entries/widget-theming.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<div>
+	<h3 id="theming">Theming</h3>
+
+	<p>The <placeholder name="name"/> widget uses the <a href="/theming/css-framework/">jQuery Mobile and UI CSS framework</a> to style its look and feel. If <placeholder name="name"/> specific styling is needed, the following CSS class names can be used for overrides or as keys for the <a href="#option-classes"><code>classes</code> option</a>:</p>
```
Comment (arschmitz) [inline entries/widget-theming.xml]: Remove `and UI`

File: entries/controlgroup.xml [hunk 2]
```diff
@@ -3,101 +3,19 @@
...
-			<type name="String" />
-		</option>
-		<option name="type" default="vertical" example-value='"horizontal"'>
-			<desc>
-				<p>Sets whether children should be stacked on top of each other or next to each other. If set to "horizontal", the children of the controlgroup will be stacked next to each other.</p>
-				<p>This option is also exposed as a data attribute: <code>data-type="horizontal"</code>.</p>
-			</desc>
-			<type name="String" />
-		</option>
-	</options>
```
Comment (arschmitz) [inline entries/controlgroup.xml]: All the deprecated options need to remain and be marked as deprecated

**Comment by arschmitz** [comment]
👍

**Comment by arschmitz** [comment]
👍

**Comment by apsdehal** [comment]
Landed in master!

**Pull request closed without merging**